omAPI/views/test_api.py

```diff
@@ -13,14 +13,26 @@
     'methods': ['GET', 'POST']
 }
 
-@app_views.route('/posts/<int:profile_id>', **settings)
-def posts(profile_id=None):
+@app_views.route('/posts/<int:profile_id>/<int:page>', **settings)
+def posts(profile_id=None, page=1):
     if request.method == 'GET':
         if profile_id is None:
             return ("Profile ID expected")
         info = storage.getPost(profile_id=profile_id)
+        print(info)
```

Comment:
I'm assuming these print statements are for testing purposes? Remember to remove these when we deploy the final version.